_posts/2019-10-01-intro-to-hmc/hmc-intro.Rmd

```diff
@@ -0,0 +1,283 @@
+---
+title: "On leapfrogs, crashing satellites, and going nuts: A very first conceptual introduction to Hamiltonian Monte Carlo"
+description: > 
+  TensorFlow Probability, and its R wrapper tfprobability, provide Markov Chain Monte Carlo (MCMC) methods that were used in a number of recent posts on this blog. These posts were directed to users already comfortable with the method, and terminology, per se, which readers mainly interested in deep learning won't necessarily be. Here we try to make up leeway, introducing Hamitonian Monte Carlo (HMC) as well as a few often-heard "buzzwords" accompanying it, always striving to keep in mind what it is all "for".
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019hmc
+date: 10-01-2019
+categories:
+  - Probability and statistics
+  - Concepts
+  - Introductions
+bibliography: bibliography.bib
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/mb.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+Why a _very (meaning: VERY!) first conceptual introduction to Hamiltonian Monte Carlo_ (HMC) on this blog? 
+
+Well, in our endeavor to feature the various capabilities of TensorFlow Probability (TFP) / [tfprobability](https://rstudio.github.io/tfprobability/), we started showing examples of how to fit hierarchical models, using one of TFP's joint distribution classes ^[[tfd_joint_distribution_sequential](https://rstudio.github.io/tfprobability/reference/tfd_joint_distribution_sequential.html)] and HMC. The technical aspects being complex enough in themselves, we never provided an introduction to the "math side of things". For those of our readers who are familiar with the technique (Stan users, for example), we hope this was acceptable; but how about those who are following the blog -- the _TensorFlow for R_ blog, alright -- due to their interest in _deep learning_? They may have felt left behind or -- put more prosaically -- just have stopped reading instantaneously.
```

Comment:
> The technical aspects being complex enough in themselves

You might phrase this so it sounds more like a qualifier (the construct: "The technical aspects being..." just sounds a little off to my ear). 

Perhaps something like: Given the complexity of the technical implementation, we never provided...